src/oAuthHelper.js

```diff
@@ -1,5 +1,5 @@
-import * as crypto from 'crypto';
 import oauthSignature from 'oauth-signature';
+import randomBytes from 'randombytes';
```

Comment:
Are you trying to make this library work from the browser?

I would have some concerns since the Bricklink API requires a client secret that should not be published to a browser.

If not, we should just use the Node runtime's crypto library.

Reply:
randomBytes is not deprecated. Maybe the import should be changed to import 'node:crypto' if there is an npm package showing up in the tool chain.
https://nodejs.org/api/crypto.html#cryptorandombytessize-callback

Reply:
Right. I (or node) couldn't find the `randomBytes` method but I think now I'll just have to install `@types/node` as dev dependency. I'll try that later tonight.